recipe/meta.yaml

```diff
@@ -24,6 +25,8 @@ requirements:
     - setuptools_scm
 
 test:
+  requires:
+    - setuptools
```

Comment:
Should `setuptools` just be a `run` requirement then? FWIW it seems `setuptools_scm` already makes `setuptools` a requirement. So this may be unnecessary.

Reply:
I added this because I explicitly used `pkg_resources` in `run_test.py`, hence this is a requirement for the test itself and independent from `setuptools_scm_git_archive`'s dependencies.
I know, I know this is overly explicit and actually covered by the package's own dependencies -- but I just don't like, by principle, to rely on implicit dependencies 😉.